index.html

```diff
@@ -9,7 +9,7 @@
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 
-    <!--
```

Comment:
try to comment your explanations correctly, because if you don't do so it will show itself in the body of the HTML page

Reply:
- [ ] Nice job mentioning the reason for the changes you asked but Kindly ask for the changes in a direct and more polite way and avoid the word `Try to`.
For this example you can say:
`Please comment your explanations correctly, because if you don't do so it will show itself in the body of the HTML page`